go/integration_test/processor/create_test.go

```diff
@@ -0,0 +1,349 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package processor
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
+)
+
+func TestProcessor_CreateAndCallContract(t *testing.T) {
+	gasLimit := big.NewInt(int64(sufficientGas))
+	gasPush := vm.PUSH1 + vm.OpCode(len(gasLimit.Bytes())-1)
+
+	for processorName, processor := range getProcessors() {
+		if strings.Contains(processorName, "floria") {
+			continue // todo implement different create types
+		}
+		for _, create := range []vm.OpCode{vm.CREATE, vm.CREATE2} {
+			t.Run(fmt.Sprintf("%s-%s", processorName, create.String()), func(t *testing.T) {
```

Comment:
Note: I realized that you can nest `t.Run` calls, and the testing infrastructure namespaces the tests for you.
this way you could t.Skip instead of continue in the processor loop 

It would be quite the same, but a little more expressive regarding tests.

Reply:
Thank you for the suggestion, I will keep this in mind for the next one. I am already working on a PR removing those lines, as they are only temporary I think they can stay in this PR.

Reply:
yes they can, you don't have to change anything.